### PR TITLE
task(auth): Control session token pruning window size

### PR DESCRIPTION
**Pull request by dschom**
## Because

- We want to be able to control the size of the session token pruning window
- The currently hard coded window size is too small.

## This pull request

- Adds a cli option, `--maxTokenAgeWindowSize`,  to configure the window size

## Issue that this pull request solves

Closes: # (issue number)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Other information (Optional)

The default has been increased from 10,000 to to 100,000; however, this may still need to be further adjusted depending on pruning response.

### CODE REVIEW

File: packages/fxa-shared/test/db/models/auth/sp_prune.sql [hunk 1]
```diff
@@ -1,7 +1,8 @@
-CREATE PROCEDURE `prune_9` (
+CREATE PROCEDURE `prune_10` (
   IN `curTime` BIGINT UNSIGNED,
   IN `maxTokenAge` BIGINT UNSIGNED,
   IN `maxCodeAge` BIGINT UNSIGNED,
+  IN `windowSize` BIGINT UNSIGNED,
```
Comment (vbudhram) [inline packages/fxa-shared/test/db/models/auth/sp_prune.sql:5]: This was easier to read than the migration file above 😅

File: packages/fxa-auth-server/scripts/prune-tokens.ts [hunk 2]
```diff
@@ -45,6 +45,11 @@ export async function init() {
       'Max age of token. Any tokens older than this value will be pruned, except session tokens with associated devices. A value of 0 results in a no-op.',
       '0'
     )
+    .option(
+      '--maxTokenAgeWindowSize <number>',
+      'The number of tokens to consider when pruning. This applies specifically to session tokens and has a limiting effect on the query. Increasing value will pull in more tokens for deletion.',
+      100000
```
Comment (vbudhram) [inline packages/fxa-auth-server/scripts/prune-tokens.ts:51]: This should be 10k right?

**Review by vbudhram — APPROVED** [review]
@dschom just the one question about default size, otw LGTM 👍🏽

File: packages/fxa-auth-server/scripts/prune-tokens.ts (hunk 2 above)
Comment (dschom) [reply packages/fxa-auth-server/scripts/prune-tokens.ts:51]: It turns out 10k wasn't nearly big enough for prod. At least that's what became apparent when we audited the production DB, so I think 100k is a good starting point, and using this as the default value _may_ result in no extra config changes on SRE's side. I don't foresee this large value having a negative impact on stage either.

**Pull request merged**